services/scan/src/precinct_scanner_app.ts

```diff
@@ -440,13 +453,26 @@ export async function buildPrecinctScannerApp(
 
   app.post<NoParams, Scan.ExportResponse, Scan.ExportRequest>(
     '/precinct-scanner/export',
-    (_request, response) => {
-      const outputStream = new streams.WritableStream();
-      store.exportCvrs(outputStream);
-      const cvrs = outputStream.toString();
+    (request, response) => {
+      const skipImages = request.body?.skipImages;
+      debug(`exporting CVRs ${skipImages ? 'without' : 'with'} inline images`);
+
+      const cvrFilename = generateFilenameForScanningResults(
+        // TODO: Move machine config provider to shared utilities and access
+        // actual machine config, with dev overrides, here instead
+        '0000',
+        getBallotsCounted(store),
+        store.getTestMode(),
+        new Date()
+      );
+
+      response
+        .header('Content-Type', 'text/plain; charset=utf-8')
+        .header('Content-Disposition', `attachment; filename="${cvrFilename}"`);
+      const cvrStream = new PassThrough();
+      cvrStream.pipe(response);
+      store.exportCvrs(cvrStream, { skipImages });
```

Comment:
Should we make this change with the central scanner app as well?

Reply:
Yeah, definitely, but I think we can punt that for NH because NH would be using the central scanner. Is that right @mattroe?

Reply:
I believe that's the case. I just thought if it's easy let's do it now. Your call.

Reply:
https://github.com/votingworks/vxsuite/issues/2682

Reply:
@carolinemodic do you want to merge this in imminently? In which case I'll put off the change and testing for central scanner.

Reply:
Yeah lets do the central scanner in a separate PR as thats not used for NH